os/include/tinyara/audio/i2s.h

```diff
@@ -213,14 +213,44 @@
 
 #define I2S_SEND(d, b, c, a, t) ((d)->ops->i2s_send(d, b, c, a, t))
 
+#define I2S_ERR_CB_RX	1
+#define I2S_ERR_CB_TX	2
+#define I2S_ERR_CB_STX	3
```

Comment:
Is this intentional, STX is 3, which sets 2 bits.

Reply:
This will be deletd. See os/arch/arm/include/s5j/i2s.h.